manifests/i/Ivanti/VelocityConsole/2.1.16.21039/Ivanti.VelocityConsole.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Ivanti.VelocityConsole
+PackageVersion: 2.1.16.21039
+Installers:
+- InstallerUrl: http://ftp.ics.cz/Zebra/Velocity/velocity_console_2.1.16.msi

```

Comment:
```suggestion
- InstallerUrl: https://www.wavelink.com/download/Downloads.aspx?DownloadFile=27577&returnUrl=/Download-Velocity_enterprise-app-modernization-Software/
```

Reply:
This package doesn't appear to be available from the publisher without registration. Packages generally have to be available for anonymous download from almost any terrestrial location at any time.